ckanext/push_errors/plugin.py

```diff
@@ -27,14 +27,17 @@ def make_middleware(self, app, config):
         def error_handler(exception):
 
             if not current_user:
-                # ignore 401, 403 and 404 errors if no user is logged in
+                # Ignorar errores específicos (401, 403 y 404) si no hay un usuario conectado
                 skip_types_if_anon = (
                     Unauthorized, Forbidden, NotFound,
                     toolkit.NotAuthorized, toolkit.ObjectNotFound,
                 )
 
-                if type(exception) in skip_types_if_anon:
-                    return
+                # Verificar tanto el tipo de excepción como su código HTTP
+                if type(exception) in skip_types_if_anon or (
+                    hasattr(exception, 'code') and exception.code in {401, 403, 404}
```

Comment:
What's `exception.code`?
Do we need this?

Reply:
https://github.com/unckan/ckanext-push-errors/pull/9/commits/341c33969b5c48e681d0fb66193d7027440f2510

Reply:
`code` is not a real property of Python `Exception` class
We don't need to monkeypatch this class to add this.
We are adding `code` to exception in oher place. We don't need to do so.
This `if` is only to skip error for anon users. As the new comment says.
I don't see why any of this changes are required. Maybe I'm missing something

Reply:
?